mp4/hvcc.go

```diff
@@ -17,9 +17,12 @@ type HvcCBox struct {
 
 // CreateHvcC - create an hvcC box based on VPS, SPS and PPS and signal completeness
 // If includePS is false, the nalus are not included, but information from sps is extracted.
-func CreateHvcC(vpsNalus, spsNalus, ppsNalus [][]byte, vpsComplete, spsComplete, ppsComplete, includePS bool) (*HvcCBox, error) {
+func CreateHvcC(vpsNalus, spsNalus, ppsNalus, seiNALUs [][]byte,
```

Comment:
I think we should remove the seiNALUs here as well, and let people add them using the AddNaluArrays method that  HvcCBox gets automatically from embedding the DecoderConfigurationRecord.

Reply:
Make sense, I updated it